.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -27,19 +22,11 @@ assignees: ''
 ## Bug Description
 <!-- Provide a clear and concise description, which will allow us to troubleshoot this bug. -->
 
-## Website Address
-<!-- 
-  If you can provide a website address, you probably have a bug with a specific website.
```

Comment:
This notice should be left-in, as before this a lot of bug reports were opened which were actually broken website.